lib/Shell/Config/Generate.pm

```diff
@@ -366,7 +366,7 @@ sub _value_escape_win32
 {
   my $value = shift() . '';
   $value =~ s/%/%%/g;
-  $value =~ s/([&^|<>()])/^$1/g;
+  $value =~ s/([&^|<>])/^$1/g;
```

Comment:
The directory with a space I have, also has parens in, similar to `Program Files (x86)`. If those get quoted, it seems the command/cmd interpreter don't like it.